modules/images/regenerate-existing-images/helper.php

```diff
@@ -0,0 +1,60 @@
+<?php
+/**
+ * Functions used by the background process API.
+ *
+ * @package performance-lab
+ * @since n.e.x.t
+ */
+
+/**
+ * Creates a background job.
+ *
+ * Job refers to the term and queue refers to the `background_job` taxonomy.
+ *
+ * @since n.e.x.t
+ *
+ * @param string $name Name of job identifier.
```

Comment:
What will that be used for? Is that the "job action"? I have been a bit confused about this throughout reviewing the PR. What is the difference between the "term name" (regular term DB field) and the name meta key we're adding here? How do we intend to use one vs the other? Please clarify. cc @jjgrainger 

In any case, I would advise choosing a more distinct name for this meta key that better describes its purpose - there is already a term name, so introducing a term meta key also called "name" is confusing.

Reply:
@ankitrox lets work together on mapping the job details to term fields/meta and determine the best way to present this.

Reply:
@felixarntz @jjgrainger Renamed it to `identifier` for better clarity.